.gitignore

```diff
@@ -136,7 +136,7 @@ bm_diff_old/
 bm_*.json
 
 # cmake build files
-/cmake/build
+**/cmake/build/
```

Comment:
isn't just `cmake/build` sufficient? (no `**/` prefix needed, anything that doesn't start with `/` will work in all subdirs too).

Reply:
I would expect that to work but I tried it and it doesn't work. My guess is that `foo/` is a shorthand for `**/foo/` because it is a single directory name, but not the abbreviation doesn't hold for paths like `**/foo/bar/` (but that's just a guess).